src/python/pants/backend/python/goals/run_python_source_integration_test.py

```diff
@@ -243,9 +243,7 @@ def test_no_strip_pex_env_issues_12057(rule_runner: RuleRunner) -> None:
     assert exit_code == 42, stderr
 
 
-@pytest.mark.parametrize("run_in_sandbox", [False, True])
-def test_pex_root_location(rule_runner: RuleRunner, run_in_sandbox: bool) -> None:
-    # See issues #12055 and #17750.
+def test_no_leak_pex_root_issues_12055(rule_runner: RuleRunner) -> None:
```

Comment:
Why aren't we tetsing both in-and-out anymore?

Reply:
What the f... some kind of merge error. Good eye!!